tap_jira/streams.py

```diff
@@ -77,9 +77,6 @@ def sync_sub_streams(page, issue_changelog_updated):
                 # when expanding changelogs for an issue, jira returns 100
                 if changelog_response['maxResults'] >= changelog_response['total']:
                     for changelog in changelogs:
-                        if utils.strptime_to_utc(changelog["created"]) > issue_changelog_updated:
```

Comment:
Removing this completely, which reverts back to previous behaviour.   The condition is inverted here.  Previously was focused on fetching more changelogs and the other branch of code was running in my test issues.

Reply:
This takes out the need to repair the issue_changelog state.  The condition is correct in the branch that fetches additional changelogs.  Its only broken when all changelogs are included